### ISSUE TITLE
[Bug] 微调做什么更新了吗？我Qlora 14B int4。上午还很正常。下午微调完全没有效果了。 着急，在线等！谢谢！

### ISSUE THREAD

**Issue by micronetboy** [QwenLM/Qwen#695]
### 是否已有关于该错误的issue或讨论？ | Is there an existing issue / discussion for this?

- [X] 我已经搜索过已有的issues和讨论 | I have searched the existing issues / discussions

### 该问题是否在FAQ中有解答？ | Is there an existing answer for this in FAQ?

- [X] 我已经搜索过FAQ | I have searched FAQ

### 当前行为 | Current Behavior

微调做什么更新了吗？我Qlora 14B int4。上午还很正常。下午微调完全没有效果了。

### 期望行为 | Expected Behavior

finetune_qlora_single_gpu.sh 使用的这个脚本微调 14B-chat-Int4。11月30日上午还很正常。下午微调完全没有效果了，微调不起任何作用了。

### 运行环境 | Environment

```Markdown
- OS:
- Python:
- Transformers:
- PyTorch:
- CUDA (`python -c 'import torch; print(torch.version.cuda)'`):
```

**Comment by micronetboy** [comment]
Loss 下降的特别快，但微调数据没有起作用，微调没有任何效果。之前是Loss太低过拟合。现在是微调没有任何效果。

**Comment by micronetboy** [comment]
我用上个版本的 finetune.py 和 modeling_qwen.py，也还是不行

微调过程中：最明显的变化是 Loss 下降特别快，微调数据不起任何作用

**Comment by fyabc** [comment]
您好，我们这边修复了`modeling_qwen.py`中的一些bug，请您拉取最新代码之后再重新尝试一下？目前本地测试Qwen-14B-Chat-Int4没有再出现loss下降过快并且微调无效的情况

**Comment by micronetboy** [comment]
感谢
